spec/support/common_actions/users.rb

```diff
@@ -81,4 +81,13 @@ def expect_to_be_signed_in
   def expect_not_to_be_signed_in
     expect(find(".top-bar-right")).not_to have_content "My account"
   end
+
+  def do_login_for(user, management)
```

Comment:
Here's a possible patch for the commit extracting this method:

```diff
diff --git a/spec/shared/system/mappable.rb b/spec/shared/system/mappable.rb
index b9e7f49e1f..04b5957e04 100644
--- a/spec/shared/system/mappable.rb
+++ b/spec/shared/system/mappable.rb
@@ -374,3 +374,7 @@ def place_map_at(latitude, longitude)
     sleep 0.01
   end
 end
+
+def do_login_for(user)
+  common_do_login_for(user, management: management)
+end
diff --git a/spec/support/common_actions/users.rb b/spec/support/common_actions/users.rb
index 6ae52d4637..55d45db75f 100644
--- a/spec/support/common_actions/users.rb
+++ b/spec/support/common_actions/users.rb
@@ -82,7 +82,7 @@ module Users
     expect(find(".top-bar-right")).not_to have_content "My account"
   end
 
-  def do_login_for(user)
+  def common_do_login_for(user, management)
     if management
       login_managed_user(user)
       login_as_manager
```

This way we don't have to pass the `management` parameter on every call :relieved:.